projects/angular-calendar/src/modules/common/util.ts

```diff
@@ -171,9 +171,12 @@ export function getWeekViewPeriod(
     viewStart = tzDateAdapter.addDays(viewStart, 1);
   }
   if (daysInWeek) {
-    const viewEnd = tzDateAdapter.endOfDay(
+    let viewEnd = tzDateAdapter.endOfDay(
       addDaysWithExclusions(tzDateAdapter, viewStart, daysInWeek - 1, excluded)
     );
+    while (tzDateAdapter.differenceInDays(viewEnd, viewStart) > 6) {
```

Comment:
Maybe we should add some comments on why we do that here. I suppose it is because of the problem with monday the 26th becoming monday the 25th when changing timezone but we could forget it.

Reply:
no this is because sometimes there is more then 7 days in a week so we have 8 column in week view when changing timezone, but indeed maybe a comment is good for that